src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -136,8 +136,7 @@ endpoint {{endpointId}} {
     emits event {{name}};
       {{/if}}
     {{/user_cluster_events}} 
-    {{#user_cluster_attributes}}
-      {{#if included}}
+    {{#enabled_attributes_for_cluster_and_side name 'server'}}
```

Comment:
This will examine attributes across all endpoints, but you want only the ones for this endpoint.  This will not do the right thing.